tools/find-em.sh.in

```diff
@@ -0,0 +1,126 @@
+#!/bin/sh
+#
+# Helper to find entries for the 'stage1 from coreos' ACI manifest files.  Uses
+# ldd and the CoreOS release contents file.  Run from within the CoreOS release
+# of interest.
+#
+# Post prosessing tips:
+#   cat found.raw | egrep '^[^=]' | egrep --invert-match '[^[0-9]\.so$' | sort | uniq -c > found.uniq
+#   manual cleanup
+#   cat found.uniq | sort | uniq > found.manifest
+#
+
+: ${BOARD:="@@BOARD@@"}
+: ${VERSION:="@@VERSION@@"}
+: ${RELEASE:="${BOARD}/${VERSION}"}
+
+contents_url="https://alpha.release.core-os.net/${RELEASE}/coreos_production_image_contents.txt"
+
+files="
+	bin/bash
+	bin/coredumpctl
+	bin/journalctl
+	bin/mount
+	bin/systemctl
+	bin/systemd-analyze
+	bin/systemd-ask-password
+	bin/systemd-cat
+	bin/systemd-cgls
+	bin/systemd-cgtop
+	bin/systemd-delta
+	bin/systemd-detect-virt
+	bin/systemd-inhibit
+	bin/systemd-machine-id-setup
+	bin/systemd-notify
+	bin/systemd-nspawn
+	bin/systemd-path
+	bin/systemd-run
+	bin/systemd-stdio-bridge
+	bin/systemd-sysusers
+	bin/systemd-tmpfiles
+	bin/systemd-tty-ask-password-agent
+	lib/systemd/system-sleep
+	lib/systemd/systemd
+	lib/systemd/systemd-ac-power
+	lib/systemd/systemd-activate
+	lib/systemd/systemd-backlight
+	lib/systemd/systemd-binfmt
+	lib/systemd/systemd-bootchart
+	lib/systemd/systemd-bus-proxyd
+	lib/systemd/systemd-cgroups-agent
+	lib/systemd/systemd-coredump
+	lib/systemd/systemd-fsck
+	lib/systemd/systemd-hostnamed
+	lib/systemd/systemd-initctl
+	lib/systemd/systemd-journald
+	lib/systemd/systemd-localed
+	lib/systemd/systemd-logind
+	lib/systemd/systemd-machined
+	lib/systemd/systemd-modules-load
+	lib/systemd/systemd-networkd
+	lib/systemd/systemd-networkd-wait-online
+	lib/systemd/systemd-random-seed
+	lib/systemd/systemd-remount-fs
+	lib/systemd/systemd-reply-password
+	lib/systemd/systemd-rfkill
+	lib/systemd/systemd-shutdown
+	lib/systemd/systemd-sleep
+	lib/systemd/systemd-sysctl
+	lib/systemd/systemd-timedated
+	lib/systemd/systemd-udevd
+	lib/systemd/systemd-update-done
+	lib/systemd/systemd-update-utmp
+	lib/systemd/systemd-vconsole-setup
+	lib/systemd/user-generators
+	bin/ip
+	bin/chmod
+	bin/chown
+	bin/ssh-keygen
+	sbin/sshd
+"
+
+download() {
+	local url=${1}
+	local file=${2}
+
+	if [ ! -f ${file} ]; then
+		echo "${url}:"
+		curl --progress-bar -o ${file} ${url}
```

Comment:
Anyway, I'd like to propose to move this script to a separate PR. I'd like this PR to merged ASAP, but the likely bikeshedding over this script will only delay it.